client/modules/IDE/components/Editor/index.jsx

```diff
@@ -254,6 +257,9 @@ class Editor extends React.Component {
       if (!prevProps.unsavedChanges) {
         setTimeout(() => this.props.setUnsavedChanges(false), 400);
       }
+    } else if (this.getContent().content !== this.props.file.content) {
+      // TODO: make this not break regular edits!
```

Comment:
@davepagurek This was pending the CM6 upgrade which will need ~2-3 weeks (after p5.js 2.0 release in any case). So we can move forward without waiting to resolve this TODO but need to keep it in mind when the upgrade is happening, @raclim will coordinate!

Reply:
I think I made this not break on the current version of CM so I think this TODO comment may be good to remove. I'll double-check this week!